src/Alerts/__tests__/ToastNotification.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { strictEqual } from 'assert';
+import {
+  render,
+  fireEvent,
+  BoundFunction,
+  GetByText,
+} from 'test-utils';
+import {
+  spy,
+  SinonSpy,
+} from 'sinon';
+import { ToastNotification } from 'Alerts';
+
+describe('Toast Notification', function () {
+  let getByText: BoundFunction<GetByText>;
+  let getByLabelText: BoundFunction<GetByText>;
+  let clickSpy: SinonSpy;
+  const id = 'test';
+  const alertMessage = 'Test Notification';
+  const buttonAltText = `${id} close button`;
+  beforeEach(function () {
+    clickSpy = spy();
+    ({ getByText, getByLabelText } = render(
+      <ToastNotification
+        id={id}
+        onClick={clickSpy}
+      >
+        Test Notification
+      </ToastNotification>
+    ));
+  });
+  afterEach(function () {
+    clickSpy.resetHistory();
+  });
+  it('renders the message', function () {
+    getByText(alertMessage);
```

Comment:
We've generally used `getByText` as a concise assertion to determine whether the component was rendered, because if `getByText` fails, it will throw an error that says that the component could not be found.